packages/unified-latex/package.json

```diff
@@ -18,19 +18,23 @@
     ],
     "exports": {
         ".": {
+            "_bundle": "./index.ts",
```

Comment:
What would happen if we just pointed to the `ts` files via `"import"`? This file gets rewritten when packaging, so that might be a solution without introducing a new condition.

Reply:
Oh, didn't know that. Just tried `"import": "./*.ts"`, and it seems that all the builds succeeds, but there's one nuisance, tests in `unified-latex-cli` invokes generated `.mjs` files directly and it needs the correct `import` values to locate imported modules. Not sure how this can be fixed.

Reply:
Hmmm...I suppose we could bundle all the deps when the cli is built. It seems reasonable that `unified-latex-cli` be a standalone thing.

Reply:
We can make `import` point to `.ts`, and `real_import` or whatever name to point whatever the previous `import` did, and call `node` with `node -C real_import` in cli testing. Then we will still be able to remove custom conditions from tsconfig and esbuild build scripts

Reply:
Would that require `real_import` to be added to every `package.json`? Either way, it seems reasonable. Let's call it `dist_import` or `prebuilt`

Reply:
Presumably yes, it should be added to every package.json that unified-latex-cli depends on